### PR TITLE
chore:(ping-centre) Ping Centre shield ID filtering should happen per ping sent.

### PR DESCRIPTION
**Pull request by emtwo**

### CODE REVIEW

**Comment by emtwo** [comment]
In https://bugzilla.mozilla.org/show_bug.cgi?id=1403695 we will be adding a "global" PingCentre instance that can be used to send general health telemetry. The first use case for it will be sending an Activity Stream health ping. 

This instance of PingCentre shouldn't always filter for the same shield IDs each time its run. But in our case when we send our A-S ping, we do want to filter for the a-s shield ID's

File: system-addon/lib/TelemetryFeed.jsm [hunk 1]
```diff
@@ -363,7 +362,8 @@ this.TelemetryFeed = class TelemetryFeed {
 
   async sendEvent(event_object) {
     if (this.telemetryEnabled) {
-      this.pingCentre.sendPing(event_object);
+      this.pingCentre.sendPing(event_object,
+      {filter: ACTIVITY_STREAM_ID});
```
Comment (Mardak) [inline system-addon/lib/TelemetryFeed.jsm]: nit: just keep this on the previous line

**Comment by as-pine-proxy** [comment]
[Treeherder: f4d0afbb1b81ae5f0e3fbc0fbe531ef06f589af7](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=f4d0afbb1b81ae5f0e3fbc0fbe531ef06f589af7)

**Comment by Mardak** [comment]
https://hg.mozilla.org/releases/mozilla-beta/rev/79375774e4fe

**Pull request merged**